runtime/src/ancient_append_vecs.rs

```diff
@@ -49,7 +49,6 @@ impl<'a> AccountsToStore<'a> {
             if available_bytes >= account_size {
                 available_bytes = available_bytes.saturating_sub(account_size);
             } else if index_first_item_overflow == num_accounts {
-                available_bytes = 0;
```

Comment:
is this removal intentional? everything else was just a rename, this seems to have a functional difference.

Reply:
no, that was left over from a previous change. Looks like it would have had no ill effect, but still incorrect.